jdaviz/configs/default/plugins/export/export.py

```diff
@@ -655,6 +696,10 @@ def save_subset_as_region(self, selected_subset_label, filename):
 
         region.write(filename, overwrite=True)
 
+    def save_subset_as_table(self, filename):

```

Comment:
Do we want an option to overwrite? Also does this need some path wrangling magic for the "standalone app"? For example, see implementation in this method:

https://github.com/spacetelescope/jdaviz/blob/ee17ed45fcbaa44931112c70c86a7e7201289fcd/jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py#L314

Reply:
Actually I see that this plugin already has `_normalize_filename` method that you wrote recently. Should that be used explicitly?

Reply:
`_normalize_filename` is already used before we get to this point, and handling overwrite is also handled in that method.

Reply:
Ah OK, hard to tell from the diff. Thanks for the clarification!